pkg/util/database/connector.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
database can be a package under pkg, no need to put it under util

Reply:
Done.